src/js/tabs/firmware_flasher.js

```diff
@@ -16,6 +16,8 @@ const firmware_flasher = {
 };
 
 firmware_flasher.initialize = function (callback) {
+    PortHandler.reinitialize(); //select recognized board
```

Comment:
This should be removed as there is already a mechanism in place for detection

Reply:
Done